tikv-client/src/main/java/com/pingcap/tikv/meta/TiTableInfo.java

```diff
@@ -155,6 +155,15 @@ TableInfo toProto() {
         .build();
   }
 
+  public boolean hasPrimaryKey() {
+    for (TiColumnInfo col : getColumns()) {
+      if (col.isPrimaryKey()) {
+        return true;
```

Comment:
maybe we should add a field value?  Once we find a primary key column, we assign the value to the field value.

Reply:
done